lib/repos/UserRepo.js

```diff
@@ -184,11 +184,18 @@ class UserRepo {
 			.collation({ locale: config.locale, numericOrdering: true });
 
 
-		let usersFromDatabase = await mongoQueryOperation.skip(start).limit(limit).toArray();
+		if (start) {
```

Comment:
Now i got this. this will fail if give 0 value as start.

Reply:
Thinking this will be fine, wouldn't `start = 0` be same as not setting skip?